vtStorAtaProtocol/Platform/Linux/ProtocolAtaPassThrough.cpp

```diff
@@ -17,174 +17,285 @@ limitations under the License.
 */
 
 #include <assert.h>
+#include <cstring>
+#include <sys/types.h>
+#include <scsi/sg.h>
+#include <sys/ioctl.h>
 
 #include "AtaProtocolEssense1.h"
-
 #include "ProtocolAtaPassThrough.h"
 
 namespace vtStor
 {
 namespace Protocol
 {
+#define SG_ATA_PASS_THROUGH_16          0x85

```

Comment:
Use static const SG_ATA_PASS_THROUGH_16 = 0x85 instead
You can read Effective C++